forgettable-frontend/src/pages/SettingsPage/SettingsPage.module.css

```diff
@@ -0,0 +1,116 @@
+.title{
+    text-align: left;
+    margin-left: 200px;
+    font-size: 40px;
+    margin-top: 65px;
+    font-weight: 600;
+    font-style: normal;
+  }
+  
+  .account{
+    text-align: left;
+    margin-left: 200px;
+    margin-top: 45px;
+    font-size: 23px;
+    font-weight: 500;
+    font-style: normal;
+  }
+  
+  .border{
+    border-style: solid;
+    color: #C4C4C4;
```

Comment:
can you instead of manually setting the colours, import the ones from the colours.css file instead?

Reply:
I have imported colours from colours.css. However, some colours cannot be imported from colours.css because there are not in colours.css. I got those colours from [here](https://www.figma.com/file/KKikShBRvD0jwnFM5aBrWP/Forgettable-UI-Design?node-id=1%3A4700) directly.

Reply:
all good then, I'll go ahead and approve these.